app/models/vim_performance_state.rb

```diff
@@ -4,6 +4,11 @@ class VimPerformanceState < ApplicationRecord
   serialize :state_data
 
   belongs_to :resource, :polymorphic => true
+  default_value_for :capture_interval, 3600
```

Comment:
I'm so bad at recognizing numbers - prefer at least having a comment, or perhaps use `1.hour.to_i`

```suggestion
  default_value_for :capture_interval, 3600 # 1 hour
```